apps/web/styles/globals.css

```diff
@@ -51,7 +51,7 @@
 .dark {
   /* background */
 
-  --cal-bg-emphasis: hsla(0, 0%, 32%, 1);
+  --cal-bg-emphasis: hsla(0, 0%, 32%,1);
```

Comment:
@sean-brydon isn't this same as before? just without space between comman and 1

Reply:
Ermmm yes - very confused as this was wrong for me locally - just merged and it’s this like someone beat me to it